utils/installer/install.sh

```diff
@@ -1,272 +1,287 @@
-#!/bin/sh
-#Set Variable to master is not set differently
-LVBRANCH="${LVBRANCH:-master}"
-USER_BIN_DIR="/usr/local/bin"
-set -o nounset # error when referencing undefined variable
-set -o errexit # exit when command fails
-
-installnodemac() {
-  brew install lua
-  brew install node
-  brew install yarn
-}
+#!/usr/bin/env bash
+set -eo pipefail
 
-installnodeubuntu() {
-  sudo apt install nodejs
-  sudo apt install npm
-}
+#Set branch to master unless specified by the user
```

Comment:
can we update this comment as well :)

Reply:
oops :)

It cannot be safely used for master until https://github.com/LunarVim/LunarVim/pull/1354 is there.